app/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "main.js",
   "scripts": {
     "start": "NODE_ENV=production startx ./node_modules/electron/dist/electron .",
-    "test": "NODE_ENV=development startx ./node_modules/electron/dist/electron ."
```

Comment:
I know you just left it the way it was, but I think `NODE_ENV` should be `test` here.